src/content/topics/integrations/observability/chronosphere.mdx

```diff
@@ -0,0 +1,74 @@
+---
+path: /integrations/chronosphere
+title: Chronosphere
+description: This topic explains how to configure and send LaunchDarkly events to
+Chronosphere.
+published: true
+tags: ['chronosphere', 'integration', 'event', 'observability']
```

Comment:
```suggestion
tags: ['chronosphere', 'integration', 'event', 'observability', 'monitoring']
```
Let's add "monitoring" here too because that's the category this integration is being added to in https://github.com/launchdarkly/integration-framework/pull/61.

(Tags do not appear in the docs, but help with ranking search results when a reader searches within docs.launchdarkly.)